README.md

```diff
@@ -9,16 +9,12 @@ virtually any problem domain, including FinTech, GovTech, and LegalTech. The Exo
 oriented towards creating permissioned blockchains, that is, blockchains with the known set of
 blockchain infrastructure providers.
 
-* [What is Exonum?](https://exonum.com/doc/get-started/what-is-exonum/)
-* [Design Overview](https://exonum.com/doc/get-started/design-overview/)
-* [Installation guide](https://exonum.com/doc/get-started/install/)
-* [Tutorial](https://exonum.com/doc/get-started/create-service/)
-* [Roadmap](https://exonum.com/doc/roadmap/)
-* [Full Specification](https://exonum.com/doc/)
-* [Reference documentation](https://docs.rs/exonum)
-* [Contributing guide](https://github.com/exonum/exonum/blob/master/CONTRIBUTING.md)
-* [Changelog](https://github.com/exonum/exonum/blob/master/CHANGELOG.md)
-
-## LICENSE
-
-Exonum core library is licensed under the Apache License (Version 2.0). See [LICENSE](https://github.com/exonum/exonum/blob/master/LICENSE) for details.
+This is the main Exonum repository that includes
```

Comment:
This does not mention the testkit, but mentions services that are not in the repo yet. If this PR does not merge these services in the core repo, I propose to describe the current state of affairs.

Reply:
This has been added mostly for the illustrative purpose. Testkit is added now, but I prefer to leave the mentioning of services and demos because they must be added before the next release (I event want to say ASAP :smile:).

Reply:
But listing components that are not actually in the repository can be misleading. Maybe, use the phrasing like "Services: configuration service (currently in [exonum-configuration repository](https://github.com/exonum/exonum-configuration))"?